admin/partials/nginx-helper-sidebar-display.php

```diff
@@ -33,12 +33,11 @@
 	<div class="inside">
 		<p>
 			<?php
-			echo wp_kses(
-				sprintf(
-					'%1$s <a href=\'%2$s\'>%3$s</a>.',
-					esc_html__( 'Please use our', 'nginx-helper' ), esc_url( 'http://rtcamp.com/support/forum/wordpress-nginx/' ), esc_html__( 'free support forum', 'nginx-helper' )
-				),
-				array( 'a' => array( 'href' => array() ) )
+			printf(
+				'%s <a href=\'%s\'>%s</a>.',
+				esc_html__( 'Please use our', 'nginx-helper' ),
+				esc_url( 'http://rtcamp.com/support/forum/wordpress-nginx/' ),
+				esc_html__( 'free support forum', 'nginx-helper' )
```

Comment:
xThis is not correct way to make small string translatable. We need to allow entire string translatable.